docs/install/index.md

```diff
@@ -105,22 +109,26 @@ npm install onnxruntime-react-native
 
 ### Install on iOS
 
-In your CocoaPods `Podfile`, add the `onnxruntime-mobile-c` or `onnxruntime-mobile-objc` pod depending on which API you wish to use.
+In your CocoaPods `Podfile`, add the `onnxruntime-c`, `onnxruntime-mobile-c`, `onnxruntime-objc`, or `onnxruntime-mobile-objc` pod, depending on whether you want to use a full or mobile package and which API you want to use.
 
```

Comment:
nit: maybe split this up a bit more clearly. 

e.g. 
In your CocoaPods `Podfile`, add the cocoapods for the onnxruntime package and API you wish to use
  - full: `onnxruntime-c`, and optionally `onnxruntime-objc`
  - mobile: `onnxruntime-mobile-c`, and optionally `onnxruntime-mobile-objc`